docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  bot:
+    image: krankkkk/dev-tj:latest
```

Comment:
Why is your name here?

Reply:
Slipped through, changed back to the `togetherjava`